kalite/distributed/features/steps/generic.py

```diff
@@ -1,6 +1,9 @@
 from behave import *
 from kalite.testing.behave_helpers import *
+from django.contrib.auth.models import User
 
 @given("I am on the homepage")
 def step_impl(context):
-    go_to_homepage(context)
\ No newline at end of file
+	if not User.objects.exists():
```

Comment:
@MCGallaspy understand the point and I like the implementation, but is there a way to make that kind of test not have to feature for ALL behave tests? I mean, the given pre-condition should be describing user behavior, too, right? So we could say that tests that aren't supposed to care about superuser creation can get away with a simpler interface?

....and then make the superuser-creation an automatic step that other behave tests automatically feature so we don't have to worry about it?